docs/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'harmony-py'
+copyright = '2021, NASA Harmony Project'
+author = 'NASA Harmony Project'
+
+# The full version, including alpha/beta/rc tags
+release = '0.0.1'
```

Comment:
Any guidance on how we'll handle release versioning? Are we going to control versions manually or have them assigned from Bamboo?

Reply:
I haven't heard anything about that.

Reply:
I'd like to use poetry or something like it which will do semantic versioning. You can then do `poetry version patch`, `poetry version minor`, `poetry version premajor`, ... And also do GitHub releases when we want to release a version.

Reply:
Just so there's record of it here, we talked about including poetry in a future PR.